### DISCUSSION TITLE
When support internationalization？

### DISCUSSION THREAD

**Discussion by caicaishmily** [TanStack/query#3099] [Ideas]
Hello, I am a developer from China. I was wondering if react-query can support internationalization ?  I'd like to translate it to chinese.

**Comment by TkDodo** [comment]
if you're talking about the docs - there seems to be a mirror page already: https://github.com/tannerlinsley/react-query/issues/1881

**Reply by ekil1100** [reply]
I see the Chinese version, it is out of date. If the official doc could support internationalization, it would be better for community growth. I can help with Chinese translation.
